backend/setupStaticContent.js

```diff
@@ -4,7 +4,7 @@ const path = require('path')
 const router = express.Router()
 
 module.exports = () => {
-  router.use(express.static(path.join(__dirname, '../build'))) // << this needed?
+  router.use('/bundle.js', express.static(path.join(__dirname, '../build/bundle.js')))
```

Comment:
only want bundle.js served here, we want to protect index.html otherwise we don't get the login prompt for react part of PSH